### PR TITLE
8332 - cleanup donate-banner.scss

### LINKED ISSUE #8332: cleanup source\sass\components\donate-banner.scss
We already use tailwind quite a bit for `network-api\networkapi\templates\partials\banner.html` and we just need to remove the sass file and replace the remaining classes

### PR DESCRIPTION
**Pull request by fessehaye**
Closes #8332 

Turn on the feature flag to view the donate banner

### CODE REVIEW

File: network-api/networkapi/templates/partials/banner.html [hunk 1]
```diff
@@ -1,8 +1,16 @@
 {% load settings_value i18n static %}
 
-{% with wrapper_class="tw-py-7" %}
-<div class="donate-banner tw-bg-black {{wrapper_class}} print:tw-hidden tw-w-full tw-hidden">
-    <a href="#" class="p-4 tw-box-content banner-close tw-cursor-pointer tw-absolute tw-right-0 tw-top-0 tw-group" aria-label="{% trans "Close Banner" %}">
+{% with wrapper_class="tw-py-7" background_class="tw-bg-cover tw-bg-center" btn_class="tw-text-white tw-border-black tw-bg-blue-40 hover:tw-bg-blue-80 tw-shadow-black"%}
+<div class="donate-banner {{background_class}} tw-bg-[url(../_images/mozilla-donate-dear-internet-mobile.png)] large:tw-bg-[url(../_images/mozilla-donate-dear-internet.png)] {{wrapper_class}} print:tw-hidden tw-w-full tw-hidden">
```
Comment (tbrlpld) [inline network-api/networkapi/templates/partials/banner.html:4]: **Question** Do we still need the `dontate-banner` class?

File: network-api/networkapi/templates/partials/banner.html [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+    <a href="#" 
+       class="p-4
```
Comment (tbrlpld) [inline network-api/networkapi/templates/partials/banner.html:6]: **Suggestion** I believe we need to use `tw-p-4` here.

File: network-api/networkapi/templates/partials/banner.html [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+              tw-box-content
+              banner-close
```
Comment (tbrlpld) [inline network-api/networkapi/templates/partials/banner.html:8]: **Question** Do we still need `banner-close`?

**Review by tbrlpld — DISMISSED** [review]

File: network-api/networkapi/templates/partials/banner.html (hunk 1 above)
Comment (fessehaye) [reply network-api/networkapi/templates/partials/banner.html:4]: we do for JS files that are targetting that class @tbrlpld

File: network-api/networkapi/templates/partials/banner.html (hunk 3 above)
Comment (fessehaye) [reply network-api/networkapi/templates/partials/banner.html:8]: used for JS file targets

**Review by tbrlpld — APPROVED** [review]

**Pull request merged**